asreview/webapp/api/projects.py

```diff
@@ -810,6 +810,44 @@ def api_random_prior_papers(project):  # noqa: F401
                 }
             )
 
+    elif subset == "unseen":
```

Comment:
This addition is nice. This function needs a refactor in the near future as the complexity level is increasing.

Reply:
Yes, indeed... but for now it seems to work :-)